spec/models/adviser_spec.rb

```diff
@@ -142,35 +144,18 @@
         adviser.run_callbacks(:commit)
       end
 
-      def queue_contains_a_job_for(job_class)
+      def aqueue_contains_a_job_for(job_class)
```

Comment:
ahh, oversight when refactoring. The method now lives in queue_spec_helper.rb. Will tidy up.